packages/ui/package.json

```diff
@@ -37,6 +37,7 @@
     "expo-blur": "^12.9.2",
     "expo-haptics": "^12.8.1",
     "expo-image-picker": "~14.7.1",
+    "fuse.js": "^7.0.0",
```

Comment:
good idea, we should probably switch from fuzzy to fuse elsewhere. looks like fuzzy is no longer maintained.

Reply:
(not in this PR tho)